.travis.yml

```diff
@@ -13,5 +13,8 @@ script:
   - bundle exec rubocop
   - bundle exec rake db:create
   - bundle exec rake db:migrate
+  - bundle exec rspec spec/models
+  - bundle exec rspec spec/controllers
+  - bundle exec rspec spec/routing
```

Comment:
These lines are no longer needed and were removed from `master` in an earlier commit. Please remove them from this file.
```
  - bundle exec rspec spec/models
  - bundle exec rspec spec/controllers
  - bundle exec rspec spec/routing
```
We don't need them any more because we have a lot of working specs now (hooray!). Running rspec on each specific directory was a good solution when we only had one directory. It was not designed to scale. Now that there are only 2 directories remaining to test, we're using an exclude pattern for those remaining directories:
```
bundle exec rspec --exclude-pattern "**/views/*/*_spec.rb, **/requests/*_spec.rb"
```